transcom/mymove
# B-19687-INT Modal issue workaround

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19687)

## Summary

Workaround for issue with our reusable modal getting caught in an infinite loop when attempting to get a feature flag. Because the evaluation report display doesn't show second/third addresses, we don't even need to get the third address feature flag. This workaround will probably not work for future issues with the modal/feature flag.

I also considered prop drilling, but preferred this solution.

### How to test

1. Login as a QAE
2. Navigate to a move
3. Click the quality assurance tab
4. Fill out the form and advance to next screen
5. Select violations and save
6. Click "View Report" and confirm you can view the report modal without anything breaking
7. Try and break
